src/content/language/methods.md

```diff
@@ -44,29 +44,30 @@ class Point {
 Operators are instance methods with special names.
```

Comment:
This isn't true for all operators in Dart. (`.`, `?.`, `..`, `||`, and `&&` aren't instance methods, for example.)

Maybe add "Most" to this sentence while you're inhere?

Reply:
+1